oecophylla/function/function.sh

```diff
@@ -2,3 +2,9 @@
 set -e
 
 conda env create --name oecophylla-humann2 -f oecophylla-humann2.yaml --quiet > /dev/null
+
+# Create the SHOGUN environment only if it hasn't already been created by the taxonomy module.
+if ! conda env list | grep -q '^oecophylla-shogun\s'
```

Comment:
Hmmm interesting point. Can you add this same code to the taxonomy.sh file so it's not order-dependent?

Reply:
Sure! Will do.